docs/developer-docs/integrations/icrc-1/icrc1-ledger-local-setup.md

```diff
@@ -0,0 +1,291 @@
+# ICRC-1 Ledger local setup
+
+## Overview
+This guide will show you how to deploy an ICRC-1 ledger locally and on mainnet, how to interact with the ICRC-1 ledger and how to test it. ICRC-1 is a token standard which you can read up on [here](https://github.com/dfinity/ICRC-1). 
+The ICRC-1 ledger used in this guide is a reference implementation. This guide aims at showing you how to setup an existing ICRC-1 ledger implementation rather than how to build an ICRC-1 ledger yourself. 
+
+### Step 1:  Make sure you use a recent version of the [IC SDK](/developer-docs/setup/install/index.mdx).
+If you don’t have the IC SDK installed, follow instructions on the [installing the IC SDK](/developer-docs/setup/install/index.mdx) section to install it.
+
+### Step 2: Create a new dfx project with the command:
+
+```
+dfx new icrc1_ledger_canister
+cd icrc1_ledger_canister
+```
+
+### Step 3:  Determine ledger file locations
+
+Go to the [releases overview](https://dashboard.internetcomputer.org/releases) and copy the latest replica binary revision. At the time of writing, this is `d87954601e4b22972899e9957e800406a0a6b929`.
```

Comment:
is this the preferred way of doing things now? this tutorial was following this approach before the approach with the shell script to download the files before, so i'm just wondering why we're changing back to it. (i prefer the non shell script approach so i'm totally fine with this change)